### ISSUE TITLE
Make code pip installable

### ISSUE THREAD

**Issue by cthoyt** [hyren/query2box#6]
It would be really useful if this code were pip-installable to make its installation and usage more reproducible. Would you be willing to accept a PR for this?

**Comment by hyren** [comment]
Yes, thanks for your efforts! Also please check out this new repo https://github.com/snap-stanford/KGReasoning with GQE, Query2box and a new BetaE model (that handles negation). Feel free to let me know if you can provide a PR to that too. Thanks :)

**Comment by cthoyt** [comment]
Hi @hyren, thanks for being patient. I've just made the PR for this repo and also I dod one for the KGReasoning repo too! Merry christmas from PR Santa :)
